### DISCUSSION TITLE
Add a DataTable Component like MUI.

### DISCUSSION THREAD

**Discussion by Abhid14** [mantinedev/mantine#1409] [Feature requests]
Hello Everyone,

I was building a UI with mantine and I encountered a situation when I needed a DataTable component. I tried making my own but it's not working out too well :( as I am a beginner. If this component exists it would be really helpful in many scenarios. Here is a DataTable from MUI https://mui.com/material-ui/react-table/#data-table if possible kindly consider adding something like this.

And thank you for this amazing library I am absolutely loving it ^ ^ .

**Comment by rtivital** [comment]
We are not planning to add this feature any time soon, use [react-table](https://react-table.tanstack.com/) to build your own
